src/components/Admin/Trainer/ControllerDataset.js

```diff
@@ -169,4 +178,20 @@ export class ControllerDataset {
 
     return {xs, ys, classes};
   }
+
+  async getUntrustedImage() {
+    const collectionReference = await this.getCollectionReference(
+      'training_data'
+    );
+    return await collectionReference
+      .where('trusted', '==', false)
+      .limit(1)
+      .get()
+      .then(snapshot => {
+        const data = snapshot.docs[0].data();
+        data.id = snapshot.docs[0].id;
```

Comment:
I'm not too sure of how this works internally but we might want to be careful adding properties to things in case they introduce weird side effects that come back to bite us!

If we just want to add the id to the object we could spread them to avoid mutating/be careful:

```
.then(snapshot => ({
  ...data,
  id: snapshot.docs[0].id
})
```